backend/src/layer/trigger/api/types/meal.rs

```diff
@@ -105,7 +105,7 @@ struct Ratings {
     /// The average rating of this meal.
     average_rating: f32,
     /// The total number of ratings for this meal.
-    ratings_count: u32,
```

Comment:
You are sadly also not allowed to change this, as this is part of the public API. Changing this would break the clients.

Reply:
Ok. I reverted it

Reply:
@Joni, could you fix the clippy warning in trigger/api/query? And maybe make it so the CI-Pipeline ignores the warnings in trigger/api/types/meal?

Reply:
How is the warning called exactly? you can put `#[allow(clippy:xxx)]` around the struct to allow ignore the warning (allow the "problem") there.